tests/scripts/multi-node/build-rook.sh

```diff
@@ -24,7 +24,10 @@ function fail_if {
 
 function purge_rook_pods {
   cd "$rook_kube_templates_dir"
+  # Older rook versions use resource type "pool", newer versions
+  # use resource type "cephblockpools.ceph.rook.io".
   kubectl delete -n rook-ceph pool replicapool || true
+  kubectl delete -n rook-ceph cephblockpools.ceph.rook.io replicapool || true
```

Comment:
If you want to keep it simpler you could just use `cephblockpools`, but this is fine if you prefer

Reply:
Do you think I should leave in the old delete line so that both types of resources are deleted or should I remove it?

Reply:
I was debating this myself... It would be nice to not include the old line but it will be fine to keep for now in case someone's still using the script with v0.8.

Reply:
Changed to `cephblockpools`